### ISSUE TITLE
Error starting framework core

### ISSUE THREAD

**Issue by julio8a** [linuxserver/docker-plex#247]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Expected Behavior
Plex shook la start up and be usable. 

## Current Behavior
Getting an error and maxed out my cpu the entire time. 

‘’’

Error starting framework core for /usr/lib/plexme
diaserver/Resources/Plug-ins-d87012962/LyricFind.
bundle.
Error starting framework core for /usr/lib/plexme
diaserver/Resources/Plug-ins-d87012962/OpenSubtit
les.bundle.
Error starting framework core for /usr/lib/plexme
diaserver/Resources/Plug-ins-d87012962/PersonalMe
dia.bundle.
‘’’

## Steps to Reproduce

1. This happened after the latest update. When I start up the container. 

## Environment
**OS:** Raspbian OS
**CPU architecture:** arm64
**How docker service was installed:**
from the official docker repo

## Command used to create docker container (run/create/compose/screenshot)
‘’’

version: "2.1"
services:
  plex:
    image: linuxserver/plex
    container_name: plex
    network_mode: host
    environment:
      - PUID=1000
      - PGID=1000
      - VERSION=docker
      - UMASK_SET=022
    volumes:
      - /home/pi/containers/plex/config:/config
      - /PiServer/Media/tv-shows:/tv-shows
      - /PiServer/Media/movies:/movies
      - /PiServer/Media/music:/music
      - /PiServer/Media/photos:/photos
      - /PiServer/Media/audio-books:/audio-books
    restart: unless-stopped
‘’’

## Docker logs

‘’’
-------------------------------------
GID/UID
-------------------------------------

User uid:    1000
User gid:    1000
-------------------------------------

[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 40-chown-files: executing... 
[cont-init.d] 40-chown-files: exited 0.
[cont-init.d] 45-plex-claim: executing... 
[cont-init.d] 45-plex-claim: exited 0.
[cont-init.d] 50-gid-video: executing... 
[cont-init.d] 50-gid-video: exited 0.
[cont-init.d] 60-plex-update: executing... 
Docker is used for versioning skip update check
[cont-init.d] 60-plex-update: exited 0.
[cont-init.d] 99-custom-scripts: executing... 
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-scripts: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
Starting Plex Media Server.
Error starting framework core for /usr/lib/plexme
diaserver/Resources/Plug-ins-d87012962/TheMovieDB
.bundle.
Error starting framework core for /usr/lib/plexme
diaserver/Resources/Plug-ins-d87012962/Fanart-TV.
bundle.
‘’’

**Comment by jwillmer** [comment]
Same(?) issue for me. After updating Plex to `1.21` it takes up all process power and returns the following errors (on Raspberry Pi 4)

```
Error starting framework core for /usr/lib/plexmediaserver/Resources/Plug-ins-d87012962/Musicbrainz.bundle.
Error starting framework core for /usr/lib/plexmediaserver/Resources/Plug-ins-d87012962/Fanart-TV.bundle.
Error starting framework core for /usr/lib/plexmediaserver/Resources/Plug-ins-d87012962/TheMovieDB.bundle.
```

**Comment by Roxedus** [comment]
I assume you are using 32bit raspi os. Use the bionic tag

**Comment by vitpachecol** [comment]
+1 testing the bionic tag now, will update when finished

**Comment by JeroenVdb** [comment]
Solve for me as well by using the bionic tag.
```
docker pull linuxserver/plex:bionic
```

Do we have to keep using this tag or can we revert to the normal one once https://github.com/GhostWriters/DockSTARTer/pull/1182 is live?

**Comment by Roxedus** [comment]
@JeroenVdb if you are using the override to set the image, you don't need to once that's live

**Comment by ipeacocks** [comment]
Sorry but still didn't get everything. Yes, I see new tag `bionic` but what does it mean in further perspective? Will that tag will be permanent? Do we need to migrate to 64bits raspbian OS (for 64bits CPU)? Any plans of deprecating of 32bits arm image?

Thanks.

**Comment by Roxedus** [comment]
Either migrate to a 64 bit os, or install a up-to-date version of libseccomp2

**Comment by jwillmer** [comment]
What is `libseccomp2`? Can you document this? My Pi does not show any open updates so in my mind I have the latest libraries.

**Comment by Roxedus** [comment]
It is a dependency of docker. As for updates, up-to-date and latest are different things. You may have the latest version pushed to the repositories you are using, however this does not mean they are up-to-date.

**Comment by jwillmer** [comment]
That means this image in not supporting Raspberry PI anymore (Only with legacy flag)?

**Comment by Roxedus** [comment]
It means that your BaseOS is outdated.

As for our support, it is not a device issue rather an OS/Package issue. If you manage to install libseccomp2 2.4 or later, it should be resolved.

**Comment by jwillmer** [comment]
Is it. I don't find any new distribution upgrade for the Pi 4:

![image](https://user-images.githubusercontent.com/1503577/100246487-7ce5ac00-2f39-11eb-8b5d-ad24b8a5fbed.png)

**Comment by Roxedus** [comment]
Since you are using buster, you can add the backport

**Comment by ipeacocks** [comment]
How base OS influences on OS inside docker? Is it some kernel stuff? Are you going to use newer base image for plex?

**Comment by Roxedus** [comment]
TL;DR The OS inside tries to use kernel calls that the host OS cant answer. 
Yes, we *need* to use Focal in order to offer full functionality in Plex. This is not the only image affected by this.

**Comment by j0nnymoe** [comment]
https://docs.linuxserver.io/faq#my-host-is-incompatible-with-images-based-on-ubuntu-focal

**Comment by julio8a** [comment]
Thanks @j0nnymoe!

**Comment by NuclearMonster** [comment]
Could the image test for this and fail instead of running at 100% CPU for weeks?
